architecture/core_objects/builds.adoc

```diff
@@ -74,7 +79,7 @@ best practices for your applications.
 
 [#custom-build]
 === Custom Build
-Custom builds allow developers to define the specific builder image responsible
-for the entire build process. The custom builder image is a plain Docker image
-with embedded build process logic, such as building RPMs or building base Docker
-images.
+link:../../creating_images/custom.html[Custom build] allows developers to define
+specific builder image, responsible for the entire build process. The custom
```

Comment:
Yeah, that's the thing I suck at :cry:, usually not enough, although recently I've started using them more often but then I got bite by overuse :wink: